ZelGray/discord-meow-bot#1: Correct a link to Reddit if a post contai…

ZelGray opened the pull request:
…ns video content

ZelGray/discord-meow-bot#2: Correct a link to TikTok

- The same do it for Twitter

soksanichenko committed:
ZelGray/discord-meow-bot#1: Correct a link to Reddit if a post contains video content
ZelGray/discord-meow-bot#2: Correct a link to TikTok

- The same do it for Twitter

soksanichenko committed:
ZelGray/discord-meow-bot#1: Correct a link to Reddit if a post contains video content
ZelGray/discord-meow-bot#2: Correct a link to TikTok

- Pylint fixes

gost-serb commented on sources/main.py:
```diff
@@ -21,6 +25,64 @@ async def ping(interaction):
+    }
+    domain = url.netloc
+    for key, value in domains.items():
+        domain = domain.replace(key, value)
```
It will cause an issue when the original link contains a fixer URL, i.e. `vxtwitter.com` -> `vxvxtwitter.com`

gost-serb commented on sources/main.py:
```diff
@@ -21,6 +25,64 @@ async def ping(interaction):
+        return
+    if not message.embeds:
+        return
+    parsed_urls = (urlparse(embed.url) for embed in message.embeds)
+    processed_urls = map(replace_domain, parsed_urls)
```
```suggestion
    processed_urls = (replace_domain(urlparse(embed.url) for embed in message.embeds)
```

soksanichenko committed:
Update sources/main.py

Co-authored-by: gost-serb <10502791+gost-serb@users.noreply.github.com>

soksanichenko committed:
ZelGray/discord-meow-bot#1: Correct a link to Reddit if a post contains video content
ZelGray/discord-meow-bot#2: Correct a link to TikTok

- Fixed a bug when an original message contains a fixed domain
- More pythonic way

soksanichenko committed:
ZelGray/discord-meow-bot#1: Correct a link to Reddit if a post contains video content
ZelGray/discord-meow-bot#2: Correct a link to TikTok

- Pylint fixes
